torch/distributed/algorithms/_checkpoint/checkpoint_wrapper.py

```diff
@@ -124,7 +143,10 @@ def checkpoint_wrapper(
             The module to be wrapped
         checkpoint_impl (Optional[CheckpointImpl]):
             The checkpointing implementation to use. Currently only
-            CheckpointImpl.REENTRANT is supported.
+            CheckpointImpl.REENTRANT is supported. Note that this will only
+            be passed into the `torch.utils.checkpoint.checkpoint`
```

Comment:
nit: I think we need double backtick for it to render as code type since this is a public API.

\`\`torch.utils.checkpoint.checkpoint\`\`

Reply:
Just as a probing question: Is the doc that says "Currently only CheckpointImpl.REENTRANT is supported." accurate still?

Reply:
Nope, we should update all this documentation.